lib/FilesHooks.php

```diff
@@ -617,7 +617,11 @@ protected function getUserPathsFromPath($path, $uidOwner) {
 
 		$path = $node->getPath();
 		$sections = explode('/', $path, 4);
-		$accessList['ownerPath'] = '/' . $sections[3];
+
+		$accessList['ownerPath'] = '/';
+		if (count($sections) === 4) {
```

Comment:
I would use an `isset($sections[3])` because it makes semantically more sense (4+ elements is fine and it actually tests, the element that is actually used later - `[0 => 'a', 1 => 'b', 2 => 'c', 4 => 'd']` is also an array of the length 4 😉  (even if it is not typical to be the case here)

Reply:
https://github.com/nextcloud/activity/pull/226/files/fbd374d889459e7148b6965cfb0d9d6690738e6a#diff-dbc5f6ba450710b214d9d73b15c9f186R619 it is max 4 elements...

Reply:
but sure

Reply:
I know, this is just about better practises and easier to understand code snippets.

Reply:
Sounds good, but yeah isset on 3 seems more logical.

Thanks for finding the issue.